src/main/java/lotto/domain/WinningNumbers.java

```diff
@@ -0,0 +1,77 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class WinningNumbers {
+    private final Lotto lotto;
+    private final LottoNumber bonusNumber;
+
+    public WinningNumbers(List<Integer> lottoNumbers, int bonusNumber) {
+        this(Lotto.from(lottoNumbers), new LottoNumber(bonusNumber));
+    }
+
+    public WinningNumbers(Lotto lotto, LottoNumber bonusNumber) {
+        validateDuplicate(lotto, bonusNumber);
+        this.lotto = lotto;
+        this.bonusNumber = bonusNumber;
+    }
+
+    private void validateDuplicate(Lotto lotto, LottoNumber bonusNumber) {
+        List<Integer> numbers = lotto.getNumbers();
+        if (numbers.contains(bonusNumber.getValue())) {
+            throw new IllegalArgumentException("보너스 번호는 당첨 번호와 중복될 수 없습니다.");
+        }
```

Comment:
```suggestion
        List<Integer> numbers = lotto.getNumbers();
        if (lotto.contains(bonusNumber)) {
            throw new IllegalArgumentException("보너스 번호는 당첨 번호와 중복될 수 없습니다.");
        }
```
lotto에서 getNumbers 사용을 지양하고 객체에 책임을 넘겨봤습니다.